packages/lab/src/accordion/AccordionDetails.tsx

```diff
@@ -43,6 +45,13 @@ export const AccordionDetails = forwardRef<
   { children, className, preventUnmountOnCollapse, ...restProps },
   ref
 ) {
+  const { window: targetWindow } = useWindow();
+  useComponentCssInjection({
+    testId: "salt-accordion",
```

Comment:
should this be  `salt-accordion-details`?

Reply:
It shouldn't because, we are injecting CSS from the file Accordion.css